test.md

```diff
@@ -0,0 +1 @@
+This file should be ignored by git
```

Comment:
According to the requirements, this file should be listed in the gitignore file in order for it not to be visible here on github. This will teach you how to make git ignore certain files in your commit.